components/org.wso2.carbon.identity.conditional.auth.functions.http/src/main/java/org/wso2/carbon/identity/conditional/auth/functions/http/AbstractHTTPFunction.java

```diff
@@ -98,6 +99,12 @@ protected void executeHttpMethod(HttpUriRequest clientRequest, Map<String, Objec
             String outcome;
             String endpointURL = null;
 
+            if (clientRequest.getMethod().equals(Constants.GET)) {
+                INVOKE_API = Constants.LogConstants.ActionIDs.INVOKE_API_HTTP_GET;
+            } else if (clientRequest.getMethod().equals(Constants.POST)) {
+                INVOKE_API = Constants.LogConstants.ActionIDs.INVOKE_API_HTTP_POST;
+            }
+
```

Comment:
Well I think you can move this to a simple method and invoke at the point the log gets logged in diagnostic log. Client request seems to be available at that point as well

Reply:
Fixed with https://github.com/wso2-extensions/identity-conditional-auth-functions/pull/161/commits/81b4b372496a3e2c4ba0778f71588e5a3416546a